<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainmenu</class>
 <widget class="QMainWindow" name="mainmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="currentuser_lbl">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>14</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="Line" name="oframe2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>14</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Session:</string>
    </property>
   </widget>
   <widget class="Line" name="oframe3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="oframe4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="oframe1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame0">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="buttonlogout">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>5</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LOGOUT</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewbook">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>350</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>VIEW BOOKING(S)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_newbook">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MAKE A NEW BOOKING</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PERSONAL INFORMATION</string>
     </property>
    </widget>
    <widget class="QLabel" name="standbymodelbl">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>SYSTEM IS NOW IN STANDBY MODE. BOOKING IN UNAVAILABLE.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancelbook">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>460</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CANCEL BOOKING(S)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_checkin">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CHECK IN</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="bookinglbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BOOKING(S)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonmainmenu">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>5</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MAIN MENU</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>761</width>
       <height>501</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QFrame" name="frame3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="infolbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PERSONAL INFORMATION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonmainmenu_2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>5</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MAIN MENU</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblphone">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>210</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Contact Number               :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblpassword">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>290</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password                         :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblfirstname">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>130</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>First Name                       :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbltitle">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>90</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Title                                :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblusername">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>250</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username                        :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbllastname">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>170</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Last Name                       :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblmileage">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>330</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total Mileage(s)               :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblttldeal">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>370</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total Transaction(s) Done:</string>
     </property>
    </widget>
    <widget class="QLabel" name="distitle">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>91</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="disFname">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>131</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="disLname">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>171</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="disphone">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>211</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="disusername">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>251</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="dispass">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>291</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="dismileage">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>331</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="disdeal">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>371</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="bookinglbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BOOKING(S)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonmainmenu_3">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>5</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MAIN MENU</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>521</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tablecancel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>761</width>
       <height>271</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>148</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>148</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>360</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel a booking by clicking on the left box.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>380</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Selection:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelselectedstatus">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>380</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancel">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>440</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>CONFIRM 
CANCELLATION</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="bookinglbl_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CHECK IN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonmainmenu_4">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>5</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MAIN MENU</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTableView" name="tablecheckin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>761</width>
       <height>441</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="timelbl">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>530</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Booking ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="bookingidlbl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>510</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Please kindly check in before the following time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Time remaining:</string>
     </property>
    </widget>
    <widget class="QLabel" name="timerlbl">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>550</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_checkedin">
     <property name="geometry">
      <rect>
       <x>634</x>
       <y>512</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>CHECK IN</string>
     </property>
    </widget>
   </widget>
   <zorder>currentuser_lbl</zorder>
   <zorder>oframe2</zorder>
   <zorder>userLabel</zorder>
   <zorder>oframe3</zorder>
   <zorder>oframe4</zorder>
   <zorder>oframe1</zorder>
   <zorder>frame3</zorder>
   <zorder>frame4</zorder>
   <zorder>frame0</zorder>
   <zorder>frame1</zorder>
   <zorder>frame5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
